frontend/src/views/Help.vue

```diff
@@ -1,9 +1,50 @@
 <template>
-    <h1>Help</h1>
+  <AppRow>
+    <AppCol class="col6">
+      <h1>General Help with MSP Direct</h1>
+
+      <h2>Using the Navigation Menu</h2>
+
+      The navigation bar towards the top of the screen displays categories of available business services.
+      <ul>
+        <li>Hovering over a category will display a dropdown of the services available in that category.</li>
+        <li>Selecting a service from the dropdown will take you to a page where you can then run a transaction associated with that service.</li>
+      </ul>
```

Comment:
We could put each of the h2/content groupings in a section if you wanted. But it doesn't really have any impact on the UI in the end.

Reply:
Updated with sections and some accessibility improvements